Kernel/VirtIO/VirtIOConsole.cpp

```diff
@@ -10,18 +11,17 @@
 namespace Kernel {
 
 unsigned VirtIOConsole::next_device_id = 0;
+const size_t VirtIOConsole::CONTROL_BUFFER_SIZE;
 
 UNMAP_AFTER_INIT VirtIOConsole::VirtIOConsole(PCI::Address address)
-    : CharacterDevice(229, next_device_id++)
-    , VirtIODevice(address, "VirtIOConsole")
+    : VirtIODevice(address, "VirtIOConsole")
+    , m_device_id(next_device_id++)
 {
     if (auto cfg = get_config(ConfigurationType::Device)) {
         bool success = negotiate_features([&](u64 supported_features) {
-            u64 negotiated = 0;
+            u64 negotiated = VIRTIO_CONSOLE_F_MULTIPORT;
             if (is_feature_set(supported_features, VIRTIO_CONSOLE_F_SIZE))
                 dbgln("VirtIOConsole: Console size is not yet supported!");
-            if (is_feature_set(supported_features, VIRTIO_CONSOLE_F_MULTIPORT))
-                dbgln("VirtIOConsole: Multi port is not yet supported!");
```

Comment:
The way we would usually do this is:
```suggestion
            u64 negotiated = 0;
            if (is_feature_set(supported_features, VIRTIO_CONSOLE_F_SIZE))
                dbgln("VirtIOConsole: Console size is not yet supported!");
            if (is_feature_set(supported_features, VIRTIO_CONSOLE_F_MULTIPORT))
                negotiated |= VIRTIO_CONSOLE_F_MULTIPORT;
```